addons/delivery/models/product_template.py

```diff
@@ -11,3 +11,4 @@ class ProductTemplate(models.Model):
         string="HS Code",
         help="Standardized code for international shipping and goods declaration. At the moment, only used for the FedEx shipping provider.",
     )
+    country_of_origin = fields.Many2one("res.country")
```

Comment:
Add a string and an helper to be crystal clear https://ec.europa.eu/taxation_customs/customs-4/international-affairs/origin-goods_en To avoid confusion with shipper country

Reply:
I am on it, thanks !